flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -303,34 +287,35 @@ public void testEnableStateLatencyTracking() throws Exception {
                             ? ((TestableKeyedStateBackend<Integer>) keyedStateBackend)
                                     .getDelegatedKeyedStateBackend(true)
                             : keyedStateBackend;
-            Assert.assertTrue(
-                    ((AbstractKeyedStateBackend<Integer>) nested)
-                            .getLatencyTrackingStateConfig()
-                            .isEnabled());
+            assertThat(
+                            ((AbstractKeyedStateBackend<Integer>) nested)
+                                    .getLatencyTrackingStateConfig()
+                                    .isEnabled())
+                    .isTrue();
         } finally {
             IOUtils.closeQuietly(keyedStateBackend);
             keyedStateBackend.dispose();
         }
     }
 
-    @Test
+    @TestTemplate
     public void testIsSafeToReuseState() throws Exception {
         CheckpointableKeyedStateBackend<Integer> backend =
                 createKeyedBackend(IntSerializer.INSTANCE);
         try {
-            Assert.assertEquals(isSafeToReuseKVState(), backend.isSafeToReuseKVState());
+            assertThat(backend.isSafeToReuseKVState()).isEqualTo(isSafeToReuseKVState());
         } finally {
             IOUtils.closeQuietly(backend);
             backend.dispose();
         }
     }
 
-    @Test
+    @TestTemplate
     public void testKeyGroupedInternalPriorityQueue() throws Exception {
         testKeyGroupedInternalPriorityQueue(false);
     }
 
-    @Test
+    @TestTemplate
     public void testKeyGroupedInternalPriorityQueueAddAll() throws Exception {
```

Comment:
```suggestion
    void testKeyGroupedInternalPriorityQueueAddAll() throws Exception {
```
could be package private